cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt

```diff
@@ -19,7 +19,7 @@ info(
 
 parameter "param_exclude_tags" do
   type "list"
-  label "Exclusion Tags Key:Value"
+  label "Exclusion Tags Key=Value"
   description "AWS tag to ignore Elastic IPs. Format: Key:Value"
```

Comment:
The description should also be updated here.

It also might make sense to use `allowed_pattern` and `constraint_description` to truly validate the input values (see [Parameter Fields](https://docs.rightscale.com/policies/developers/reference/v20180301/policy_template_language.html#parameters-fields)).